src/Parser/BloodDeathKnight/Modules/Items/T20_4pc.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const RuinicDiscound=5;
```

Comment:
Magic number constants should be all caps snake style; e.g. GRAVEWARDEN_RUNIC_DISCOUNT

Reply:
fixed